app/views/articles/show.html.erb

```diff
@@ -0,0 +1,41 @@
+
+<% if can? :update, @article %><%= link_to 'Edit', edit_article_path(@article) %> | <% end %> 
+<%= link_to 'Back', articles_path %>
+<p>
+  <strong>Title:</strong>
+  <%= @article.title %>
+</p>
+ 
+<p>
+  <strong>Text:</strong>
+  <%= @article.body %>
+</p>
+ 
+ <hr>
+<h2 class="text-center">Comments</h2>
+
+<%= form_with(model: [ @article, @article.comments.build ], local: true) do |form| %>
+    <div class="row d-flex justify-content-center">
+        <div class="col-10">
+            <div class="comment-box ml-2">
+                <h4>Add a comment</h4>
+                <%= form.hidden_field :commenter, :value => User.current_user.role_id == User::ROLES[:Admin] ? "Admin" : User.current_user.name %>
```

Comment:
User.current_user.role_id == User::ROLES[:Admin] ? "Admin" : User.current_user.name move this to a helper method.